nss-cohort-37/Workforce-Red-Jaguars
# Employees can now be edited

# Description
1) Employees are now able to be edited.
2) Dropdowns of both Departments and Employees should appear in the edit
3) Only unassigned computers can be assigned to an editable employee, including their currently assigned computer

Fixes # (issue)
## Type of change
Please delete options that are not relevant.

# Testing Instructions
Check if an employee can be edited and if the dropdown only lists available computers. Edit once more to see if the newly selected computer is automatically selected and if other assigned computers are removed from the list. 

# Checklist:
